tests/finding_test.py

```diff
@@ -51,6 +53,36 @@ def test_list_findings(self, suffix):
         self.assertEqual(bulk_edit_menu.find_element_by_id("id_bulk_out_of_scope").is_enabled(), True)
         self.assertEqual(bulk_edit_menu.find_element_by_id("id_bulk_is_mitigated").is_enabled(), True)
 
+    def test_quick_report(self):
+        # bulk edit dropdown menu
+        driver = self.driver
+        driver.get(self.base_url + "finding")
+
+        driver.find_element_by_id("downloadMenu").click()
+        driver.find_element_by_id("report").click()
+
+        self.assertIn("<title>Finding Report</title>", driver.page_source)
+
+    def test_csv_export(self):
+        driver = self.driver
+        driver.get(self.base_url + "finding")
+
+        driver.find_element_by_id("downloadMenu").click()
+        driver.find_element_by_id("csv_export").click()
+
+        time.sleep(2)
+        self.assertTrue(Path("/tmp/findings.csv").is_file())
+
+    def test_excel_export(self):
+        driver = self.driver
+        driver.get(self.base_url + "finding")
+
+        driver.find_element_by_id("downloadMenu").click()
+        driver.find_element_by_id("excel_export").click()
+
+        time.sleep(2)
+        self.assertTrue(Path("/tmp/findings.xlsx").is_file())
```

Comment:
I think this is a server side assert? And I guess you looked at doing a client side quick, but it's not possible or hard?

Reply:
or maybe it is client side, just saw the prefs now above

Reply:
What do you mean with client or server side assert, is there a difference? The implementation comes from the Python `unittest` package and raises an exception when the expression is false:

```
    def assertTrue(self, expr, msg=None):
        """Check that the expression is true."""
        if not expr:
            msg = self._formatMessage(msg, "%s is not true" % safe_repr(expr))
            raise self.failureException(msg)
```

Reply:
In some unit tests there is sometimes a mix of things being checked. In the api tests for example some asserts are on the data returned to the client (so "client-side") and some asserts are on server-side data like things in the database.
At first glance the `/tmp/findings.xlsx` assert looked like "server-side" to me. But I was wrong, so forget about it :-)